src/module.c

```diff
@@ -2974,6 +2974,10 @@ int RM_ZsetRangePrev(RedisModuleKey *key) {
  *                          are created.
  *     REDISMODULE_HASH_CFIELDS: The field names passed are null terminated C
  *                               strings instead of RedisModuleString objects.
+ *     REDISMODULE_HASH_COUNT_INSERTS: Include the number of inserted fields in
+ *                                     the returned number, in addition to the
+ *                                     number of updated and deleted fields.
+ *                                     (Added in Redis 6.2.)
```

Comment:
i think the term "inserted" is confusing ("updated" is confusing too).

"updated" in this case means a pre-existing member was updated.
but i think the term can also mean new field that was added (the hash object was updated).
so i think we should make it clear in the docs.

similarly, "inserted" sounds like it only means new fields (excluding pre-existing fields), but the code actually means any field that is "set" (both new, and updates). let's try to come up with a better term, and if not, at least make it very clear in the docs.

Reply:
Oh, I interpreted it as "inserts" meaning only new fields, and the new flag COUNT_INSERTS means "count the inserts in addition to the updates".

So yeah, it seems these docs need to be improved. We could also rename the flag. Any suggestion?

Reply:
@itamarhaber maybe you can come up with a term?

Reply:
COUNT_ALL? (meaning inserted, updated/replaced and deleted fields)

Reply:
ok.. better than INSERTS. IMHO. or we can wait for other suggestions.
p.s. there are other references to "updates" and "inserts" in the text, we need to make sure they don't get out of context.